jujugui/static/gui/src/app/components/deployment-bar-change-count/_deployment-bar-change-count.scss

```diff
@@ -0,0 +1,14 @@
+.deployment-bar__change-count {
+    display: inline-block;
+    margin-right: 1px;
+    padding: 10px 15px;
+    background-color: $mid-grey;
+    border-top-left-radius: $border-radius;
+    border-bottom-left-radius: $border-radius;
+    color: $white;
+    transition: background .3s;
+
+    &--active {
+        background-color: $uncommitted;
+    }
```

Comment:
Personally I would put all the `.deployment-bar__change-count` into a placeholder and then
```sass
.deployment-bar__change-count {
    @extend %deployment-bar__change-count;

    &--active {
        @extend %deployment-bar__change-count;
        background-color: $uncommitted;
    }
}
```
This would mean you could toggle the classes deployment-bar__change-count and deployment-bar__change-count--active instead of having both classes.